stories/PInput.stories.js

```diff
@@ -0,0 +1,37 @@
+import { action } from '@storybook/addon-actions';
+import PInput from '../src/components/PInput.vue';
+import Render from './Render';
+
+export default {
+  title: 'PInput',
+};
+
+export const basicUsage = () => Render({
+  components: { PInput },
+  template: '<PInput @focus="action" name="test_input" value="">',
```

Comment:
`value` prop should actually be v-model. [Read more about it](https://vuejs.org/v2/guide/components.html#Using-v-model-on-Components) then please update **all** examples.

The please put an `@input` just to log changes on v-model